src/common/network/NetworkUtils.cpp

```diff
@@ -114,13 +114,56 @@ bool NetworkUtils::getDynamicPortRange(uint16_t& low, uint16_t& high) {
 
```

Comment:
Could you update [`.github/workflows/ci.yaml`](https://github.com/vesoft-inc/nebula/blob/master/.github/workflows/ci.yaml#L32) and add `-j$(nproc)` option to `ctest --output-on-failure` command

Reply:
sure